MobileCenterDistribute/MobileCenterDistribute/Resources/fr.lproj/MobileCenterDistribute.strings

```diff
@@ -3,3 +3,4 @@
 "Download" = "Télécharger";
 "Update available" = "Mise à jourr disponible";
 "No release notes" = "Aucune Release Note n'est fournie pour cette version.";
```

Comment:
Now that I see this one I would change it for "Aucune note de version n'est fournie."